src/NuGet.Services.Staging.BackgroundWorkers/project.json

```diff
@@ -7,13 +7,13 @@
         "NuGet.Services.Staging.PackageService": "1.0.0-*",
         "Microsoft.Extensions.Options": "1.0.0-rc2-*",
         "Newtonsoft.Json": "8.0.3",
-        "NuGet.Resolver": "3.5.0-*",
+        "NuGet.Resolver": "3.5.0-beta-final",

```

Comment:
using * to get the newest NuGet.* packages breakes me twice a day.. and also:  https://github.com/aspnet/EntityFramework/issues/5411